.buildkite/scripts/common/env.sh

```diff
@@ -83,6 +83,11 @@ if is_pr; then
     export ELASTIC_APM_CONTEXT_PROPAGATION_ONLY=true
   fi
 
+  # value for security genai prompts evals
+  if is_pr_with_label "ci:security-genai-prompts-run-evals"; then
```

Comment:
nit: Maybe tweak name?  

`ci:security-genai-run-evals-default-prompts`
`ci:security-genai-run-evals-local-prompts`
`ci:security-genai-run-evals-reset-prompts`
`ci:security-genai-run-evals-clear-prompts`


Keeps the same root as the other label and a little more descriptive. Though we can add description text to the label in GitHub, so not a big deal either way.